zscaler/icon_zscaler/actions/lookup_url/action.py

```diff
@@ -13,7 +14,15 @@ def __init__(self):
                 output=LookupUrlOutput())
 
     def run(self, params={}):
-        lookup_urls = params.get(Input.URLS)
+        urls = params.get(Input.URLS)
+        lookup_urls = []
+        for url in urls:
+            if url and not url.startswith("http"):
+                url = f"http://{url}"
```

Comment:
We should lookup both the https and http version of the URL (2 URLs)

Reply:
Lookup and blacklist works only for domain. Not url. So we add here http to work with urlparse which get domain part from url. Scheme is not passing to API.

```
lookup_urls.append(
    urlparse(url).hostname
)
```

Reply:
What if URL is https://www.google.com?

Reply:
``` 
urls: ['https://exampletests123.com', 'http://exampletests1234.com', 'exampletests12345.com']
normalized_urls: ['exampletests123.com', 'exampletests1234.com', 'exampletests12345.com']
Starting new HTTPS connection (1): admin.zscalerbeta.net:443
https://admin.zscalerbeta.net:443 "POST /api/v1/authenticatedSession HTTP/1.1" 200 138
Starting new HTTPS connection (1): admin.zscalerbeta.net:443
https://admin.zscalerbeta.net:443 "POST /api/v1/security/advanced/blacklistUrls?action=ADD_TO_LIST HTTP/1.1" 204 0
{"body": {"log": "Connect: Connecting...\nrapid7/Zscaler:1.1.2. Step name: blacklist_url\nurls: ['https://exampletests123.com', 'http://exampletests1234.com', 'exampletests12345.com']\nnormalized_urls: ['exampletests123.com', 'exampletests1234.com', 'exampletests12345.com']\n", "status": "ok", "meta": {}, "output": {"success": true}}, "version": "v1", "type": "action_event"}
``` 
Schema is unused :) I added logger to check those situations :)